### PR TITLE
fix(select): Removing buggy animation of the arrow in the default select - FRONT-4348

### PR DESCRIPTION
**Pull request by planctus**

### CODE REVIEW

**Comment by emeryro** [comment]
not sure what is the issue here, but on my side the arrow animation still doesn't work as expected.
The arrow only rotate while I'm holding the mouse click, and goes back to default when I release it (regardless of the open/close status).
With the keyboard it's better, but still the arrow doesn't goes back to default when pressing escape

**Comment by planctus** [comment]
The issue is that currently, if you click outside of the select to close it, the animation doesn't work anymore, in this pull request this is fixed, but it's true that we cannot fully handle the first usage of Esc, which results in closing the dropdown without animating the select, but this is a minor issue to me, considering that when you move the focus out the arrow will go back to its default position.
I read what you described about the behaviour, i do not see that, for me it works fine using the mouse, you click it opens the dropdown and rotates the icon, clicking again will close the dropdown and rotate again the icon.

Aside of this, why do we have this huge difference in the icon size for the single and the multiple select now..?

**Comment by emeryro** [comment]
I don't know if we are talking about the same thing, but on my side the arrow animation doesn't work at all with the mouse (unless I keep the button pressed). Clicking outside the select or not does no difference.
simple click
2024-04-02-select-click

"long" click
2024-04-02-select-click-long

For the arrow styling, I don't know... They are indeed quite different, but I don't know if that was intentional. We'll see with the designers tomorrow

**Comment by planctus** [comment]
Tried a bit more, but it seems really impossible to produce a logic that would work for all the browsers and it's also confirmed by the literature on internet, so i removed the animation from the default select.

File: src/implementations/vanilla/components/select/select.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ const iconSize = system === 'eu' ? 's' : 'xs';
 
 /**
  * This API mostly refers to the multiple select, in the default select only three methods are actually used:
```
Comment (emeryro) [inline src/implementations/vanilla/components/select/select.js]:
```suggestion
 * This API mostly refers to the multiple select, in the default select only two methods are actually used:
```

**Review by emeryro — CHANGES_REQUESTED** [review]
Now I have a js error when trying to open the select with the keyboard:
image

**Pull request merged**